docs/commands/build-docs.md

```diff
@@ -0,0 +1,73 @@
+# `build-docs`
+
+## Introduction
+
+The `build-docs` build api and Redoc into a zero-dependency HTML file.
+
+## Usage
+
+```bash
+redocly build-docs <api>
+redocly build-docs <api> --output=custom.html
+redocly build-docs <api> --options.theme.colors.primary.main=orange
+redocly build-docs <api> -t custom.hbs
+redocly build-doc <api> -t custom.hbs --templateOptions.metaDescription "Page meta description"
+```
+
+
+## Options
+
+Option | Type | Description
+-- | -- | --
+api | string | **REQUIRED.** path or URL to your to the API definition file.
```

Comment:
Does it work to pass in the name from the configuration file:

```
apis:
  apiname@v1:
    root: ...
```

`redocly build-docs apiname@v1`?

Reply:
no, it doesn't work for now. We should improve that later. 
Firstly we want to deprecate `redoc-cli`.
In the first version of this PR, it contains `<spec>`.
I talked with @RomanHotsiy , and he recommended renaming `<spec>` to `<apis>` for docs consistency.
What do you think about that? Should we rename that to `<spec>`?

Reply:
I think this introduces inconsistency into commands actually. Please hold off on doing this. 

All commands should accept `apis` which is a list of file paths or aliases from the configuration file.